src/Processor.java

```diff
@@ -1,40 +1,61 @@
 
- class Processor {
-	
+class Processor {
+
 	private Car[] carProc;
-	
-	public Processor(Car[] carProc) {
-		this.carProc = carProc;
-	}
-	
-	void findeCarByMark(String m) {
-		for(Car car:carProc) {
-			if(m.equals(car.getMarka()))
+
+	public Processor() {
+
+	}
+
+	public Processor(Car[] car) {
+		this.carProc = car;
+	}
+
+	public void findeCarByMark(Car[] cars, String marka) {
+		for(Car car : cars) {
+			if(marka.equals(car.getMarka()))
```

Comment:
It's bad approach to use if without brackets it may lead you to unexpected bug in the future.